docker-compose-dev.yaml

```diff
@@ -39,14 +44,18 @@ services:
 
       MY_EDC_PROTOCOL: "http://"
       MY_EDC_FQDN: "edc"
-      MY_EDC_IDS_BASE_URL: "http://edc:11003"
+      MY_EDC_PROTOCOL_BASE_URL: "http://edc:11003"
 
       MY_EDC_JDBC_URL: jdbc:postgresql://postgresql:5432/edc
       MY_EDC_JDBC_USER: edc
       MY_EDC_JDBC_PASSWORD: edc
       EDC_WEB_REST_CORS_ENABLED: 'true'
       EDC_WEB_REST_CORS_HEADERS: 'origin,content-type,accept,authorization,x-api-key'
       EDC_WEB_REST_CORS_ORIGINS: '*'
+
+      EDC_PARTICIPANT_ID: provider
```

Comment:
should be moved to .env and sourced from MY_EDC_ID_KEBAP_CASE or whatever it's called